source/_integrations/clicksend_tts.markdown

```diff
@@ -13,42 +13,40 @@ ha_integration_type: integration
 
 The `clicksend_tts` platform uses [ClickSend](https://clicksend.com) to deliver text-to-speech (TTS) notifications from Home Assistant.
 
-Go to your [ClickSend Dashboard](https://dashboard.clicksend.com) section and create your new project. After creating your project, you should now be able to obtain your `username` and `api_key`.
+After creating your account, you should now be able to obtain your `username` and `api_key` [here](https://dashboard.clicksend.com/account/subaccounts).
 
 To add ClickSend to your installation, add the following to your Home Assistant `configuration.yaml` file:
 
 ```yaml
 notify:
   - platform: clicksend_tts
-    username: CLICKSEND_USERNAME
-    api_key: CLICKSEND_API_KEY
-    recipient: PHONE_NO
+    username: test@example.com
+    api_key: D83DED51-9E35-4D42-9BB9-0E34B7CA85AE
+    language: fr-fr
+    voice: male
+    recipient: +61411111111
```

Comment:
We try to keep all configuration samples minimal. Thus, no optional requirement in the default sample. This helps a user to get started quickly by copy-&-paste the sample without worrying about optional parameters which they most likely not need. If required, insert a full configuration sample later that covers special setups or alike.

Reply:
Ok I will remove the optional requirements